hibernate-redis/src/main/java/org/hibernate/cache/redis/jedis/JedisClient.java

```diff
@@ -432,6 +433,88 @@ public void execute(Transaction tx) {
     }
 
     /**
+     * We want nextTimestamp to return a long that is greater than previous calls to nextTimestamp.
+     *
+     * The simplest algorithm would be to ignore Redis and return currentTimeMillis. However, different VMs can have small drifts
```

Comment:
@mrniko I am pretty new to Redisson and lua, would you have time to offer an example or further pointers?